lib/Nodes/Table/TableColumn.php

```diff
@@ -64,10 +64,19 @@ public function setNode(Node $node) : void
 
     public function render() : string
     {
-        return $this->getNode()->render();
+        $rendered = $this->getNode()->render();
+
+        if ('' === $rendered && '\\' !== $this->content) {
+            $rendered = '&nbsp;';
```

Comment:
This is the odd-ball. If a column is empty, then we put in an `&nbsp;`... unless the original content was `\`, in which case this means "please keep my column, but make it fully empty, no space"